### ISSUE TITLE
Entered content vanishes after popup close.

### ISSUE THREAD

**Issue by needfulthing** [arlogn/easyshare#21]
I juts tried out your plug-in. But when you enter content and for any reason the popup closes and you re-open it, all content is gone :(

**Comment by arlogn** [comment]
The popup's content is loaded every time the popup is shown, and unloaded every time the popup is closed. This is by design in the webextensions APIs. The feature you ask presents some problems and I don't know if it can be introduced. However, I put it on the TODO list.

**Comment by needfulthing** [comment]
Thanks for your reply :) Could storage.local be used to cache the content?

**Comment by arlogn** [comment]
Yes, the content must be saved when the popup unload. But only if it isn't sent. And you must check that it is not a notification.

**Comment by Pascal-KOTTE** [comment]
oops, sorry, My #22 is the same issue this one

**Comment by RoyTheNewbie** [comment]
In the online video there is a SAVE button in the publisher. But that button is not present now. Is there now no way of saving content sequentially, as shown in the video?

**Comment by arlogn** [comment]
The installable version is 0.16.2. The feature you are talking about is present in version 0.17.0 which you can download from github. As I have already had the opportunity to tell you I cannot update the addon on AMO due to a bug that made my old account inaccessible. You can test the version in question by installing it in debugging mode in Firefox: 
`about:debugging > This firefox > Load temporary addon` and select the manifest.json file.
I added a note on the [web page](https://arlogn.github.io/easyshare/).

**Comment by RoyTheNewbie** [comment]
I am very happily using version .17.0, following the instructions you provided. I just wanted to thank you because this version is far superior to 0.16.2 in terms of its efficiency and usability. I would like to make on suggestion for the future, but it may not be possible. Namely, I find that every time I use the Easyshare addon I need to change "All Aspects" to "Public". It would be good to have a user setting for that, so that this preference was shown as the default. Maybe there are technical reasons why that cannot be done.
_Thanks again! I certainly hope that you will maintain this valuable addon._

**Comment by arlogn** [comment]
Hi, as soon as I can I will check if it's possible to insert a selection in the preferences. The problem is that I think it's a bit cumbersome to do. I have to check.
